rca/account_management/views.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.auth.models import Group
+from django.views.generic import FormView
+
+from rca.users.models import User
+
+from .forms import StudentCreateForm
+
+
+class CreateStudentFormView(FormView):
+
+    template_name = "account_management/create.html"
+    form_class = StudentCreateForm
+    success_url = "/admin/student/create"  # TODO - better success template
+
+    def form_valid(self, form):
+        data = super().form_valid(form)
+        student_group = Group.objects.get(name="Students")
+        student_user = User.objects.create_user(
+            username=form.cleaned_data["username"],
+            password="123pass",
+            email=form.cleaned_data["email"],
+            first_name=form.cleaned_data["first_name"],
+            last_name=form.cleaned_data["last_name"],
+        )
+        student_user.groups.add(student_group)
```

Comment:
At the moment, as part of the previous ticket, adding this group here will initiate the code in this signal https://github.com/torchbox/rca-wagtail-2019/blob/728-student-profile-student-journey/rca/users/signals.py

I think it might be worth moving the code in that signal to create page into this `form_valid()` method, along with a manual option to 'create_student_page` on the form?

Reply:
If this is going to be the only mechanism by which student users are created, I'd be inclined to agree that this `form_valid()` method would be a good place for the code in the signal.

> along with a manual option to 'create_student_page` on the form?

Can you explain a bit more what you mean here? As in, there'd be a field on the form which could control whether a student page was created? Or something else (e.g. a supplementary form for adding a student page to an existing user)?

Reply:
Yea my thoughts here are to implement a simple boolean on the form to give more control over whether a page should be created